lib/dialogs/settings/index.jsx

```diff
@@ -141,6 +142,9 @@ export class SettingsDialog extends Component {
         <div className="dialog-column">
           <DisplayPanel buckets={buckets} />
         </div>
+        <div className="dialog-column">
```

Comment:
Should we move these `dialog-column` divs to within the panel components? 🤔

Reply:
True, the CSS is a bit messy! I refactored the TabPanels component so the CSS isn't all over the place.